.github/workflows/algolia.yml

```diff
@@ -0,0 +1,9 @@
+name: Algolia crawler creation and crawl
```

Comment:
@dangerismycat is this a correct GitHub workflow format? E.g. when is this action supposed to execute?

I think we might need something along these lines:
https://github.com/algolia/algoliasearch-crawler-github-actions/blob/main/examples/basic.yml
(perhaps only re-crawl when we push to `master`?)

Reply:
Probably not! I have no idea what I'm doing here. 😅

Thanks for passing along that example. I'll pattern-match from that!